src/Main.ts

```diff
@@ -79,35 +79,35 @@ Object.freeze(window.tsParticles);
 
 /* particles.js compatibility */
 window.particlesJS = (tagId: string, params: IOptions) => {
-  if (console) {
-    console.info("this method is obsolete, please use the new tsParticles.load");
-  }
+    if (console) {
+        console.info("this method is obsolete, please use the new tsParticles.load");
+    }
 
-  window.tsParticles.load(tagId, params);
+    return window.tsParticles.load(tagId, params);
 };
 
-window.particlesJS.load = async (tagId: string, pathConfigJson: string, callback: () => void) => {
-  if (console) {
-    console.info("this method is obsolete, please use the new tsParticles.loadJSON");
-  }
+window.particlesJS.load = (tagId: string, pathConfigJson: string, callback: (container: Container | undefined) => void) => {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Exceeds maximum line length of 120](https://app.codacy.com/manual/ar3s/tsparticles/pullRequest?prid=5036765)